metaflow/cli.py

```diff
@@ -807,12 +752,36 @@ def start(ctx,
                                                   ctx.obj.flow,
                                                   ctx.obj.event_logger,
                                                   ctx.obj.monitor)
-    ctx.obj.datastore = DATASTORES[datastore]
+
+    ctx.obj.datastore_impl = DATASTORES[datastore]
 
     if datastore_root is None:
         datastore_root = \
-          ctx.obj.datastore.get_datastore_root_from_config(ctx.obj.echo)
-    ctx.obj.datastore_root = ctx.obj.datastore.datastore_root = datastore_root
+          ctx.obj.datastore_impl.get_datastore_root_from_config(ctx.obj.echo)
+    if datastore_root is None:
+        raise CommandException(
+            "Could not find the location of the datastore -- did you correctly set the "
+            "METAFLOW_DATASTORE_SYSROOT_%s environment variable" % datastore.upper())
```

Comment:
Nit: Leave a "?" at the end of the mesage.

Reply:
OK, will fix.

Reply:
Done